packages/shared/src/lib/localLanguage.ts

```diff
@@ -1,50 +1,41 @@
 export const LocalLanguageBannerTemplateName = 'EuropeMomentLocalLanguageBanner';
```

Comment:
It looks like the `LocalLanguageBannerTemplateName` is being used once in the banner router so I'd suggest removing this var in favour of using the Europe banner module's name as a string in the router

Reply:
Yes changed